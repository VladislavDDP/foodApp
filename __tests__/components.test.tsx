import React from 'react';
import renderer from 'react-test-renderer';

import {LoadingScreen} from '../src/components/loading-screen/LoadingScreen.component';

describe('Components rendering', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<LoadingScreen title="test" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
