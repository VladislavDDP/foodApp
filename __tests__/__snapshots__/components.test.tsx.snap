// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components rendering renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#999",
        "height": "100%",
        "justifyContent": "center",
        "left": 0,
        "position": "absolute",
        "top": 0,
        "width": "100%",
        "zIndex": 10,
      },
      Object {
        "backgroundColor": "#36153b",
      },
    ]
  }
>
  <ActivityIndicator
    size="large"
    style={
      Array [
        Object {
          "backgroundColor": "#373737",
        },
        Object {
          "backgroundColor": "transparent",
        },
      ]
    }
  />
  <Text
    style={
      Array [
        Object {
          "color": "#fff",
        },
        Object {
          "fontSize": 25,
          "fontWeight": "700",
          "marginTop": 10,
        },
      ]
    }
  >
    test
  </Text>
</View>
`;
